src/detector.jl

```diff
@@ -42,6 +42,11 @@ Look up fieldgen generated electric field and weighting potential files
 function simulate_detector(det_meta::PropDict, env::Environment, simulator::SiggenSimulator;
         overwrite::Bool = false)
 
+    # if no cached name given, force simulation from scratch (or else might read past "tmp" file)
+    if simulator.cached_name == ""
+        overwrite = true
+    end
```

Comment:
Maybe add a `warning` or `info` here, that the keyword `overwrite` will be overwritten (?)

Reply:
Yeah this part with messages should all change. For now the main @info messages happen in `PSSimulator`, which makes it print the same @info two times confusing the user when SSD geometry is constructed to remove events outside of the detector in `pet_to_stp()` :') So now in `PSSimulator` it says "No cached name provided, not caching the simulation". I'll get on cleaning up the messages at some point.